### ISSUE TITLE
Update OkHttp version 

### ISSUE THREAD

**Issue by nuwanchamara** [amitshekhariitbhu/Fast-Android-Networking#580]

**Comment by nuwanchamara** [comment]
Can you update OkHttp version and make a release ?

**Comment by Inside4ndroid** [comment]
agreed we need okhttp updating to fix an issue on newer versions of android...

`2022-02-09 15:53:22.673 32175-32286/com.htetznaing.Jresolverexample W/resolverexampl: Accessing hidden method Lcom/android/org/conscrypt/ConscryptEngineSocket;->setUseSessionTickets(Z)V (max-target-q,core-platform-api, reflection, denied)
2022-02-09 15:53:22.673 32175-32286/com.htetznaing.Jresolverexample W/resolverexampl: Accessing hidden method Lcom/android/org/conscrypt/OpenSSLSocketImpl;->setUseSessionTickets(Z)V (max-target-q,core-platform-api, reflection, denied)
2022-02-09 15:53:22.673 32175-32286/com.htetznaing.Jresolverexample W/resolverexampl: Accessing hidden method Lcom/android/org/conscrypt/AbstractConscryptSocket;->setUseSessionTickets(Z)V (max-target-q, reflection, denied)
2022-02-09 15:53:22.673 32175-32286/com.htetznaing.Jresolverexample W/resolverexampl: Accessing hidden method Lcom/android/org/conscrypt/ConscryptEngineSocket;->setHostname(Ljava/lang/String;)V (max-target-q,core-platform-api, reflection, denied)
2022-02-09 15:53:22.673 32175-32286/com.htetznaing.Jresolverexample W/resolverexampl: Accessing hidden method Lcom/android/org/conscrypt/OpenSSLSocketImpl;->setHostname(Ljava/lang/String;)V (max-target-q,core-platform-api, reflection, denied)
2022-02-09 15:53:22.673 32175-32286/com.htetznaing.Jresolverexample W/resolverexampl: Accessing hidden method Lcom/android/org/conscrypt/AbstractConscryptSocket;->setHostname(Ljava/lang/String;)V (max-target-q, reflection, denied)
2022-02-09 15:53:22.673 32175-32286/com.htetznaing.Jresolverexample W/resolverexampl: Accessing hidden method Lcom/android/org/conscrypt/OpenSSLSocketImpl;->setAlpnProtocols([B)V (max-target-q,core-platform-api, reflection, denied)
2022-02-09 15:53:22.673 32175-32286/com.htetznaing.Jresolverexample W/resolverexampl: Accessing hidden method Lcom/android/org/conscrypt/AbstractConscryptSocket;->setAlpnProtocols([B)V (max-target-q, reflection, denied)
2022-02-09 15:53:22.743 32175-32286/com.htetznaing.Jresolverexample W/resolverexampl: Accessing hidden method Lcom/android/org/conscrypt/OpenSSLSocketImpl;->getAlpnSelectedProtocol()[B (max-target-q,core-platform-api, reflection, denied)
2022-02-09 15:53:22.744 32175-32286/com.htetznaing.Jresolverexample W/resolverexampl: Accessing hidden method Lcom/android/org/conscrypt/AbstractConscryptSocket;->getAlpnSelectedProtocol()[B (max-target-q, reflection, denied)`

**Comment by Bennik2000** [comment]
I also need a new version with updated OkHttp
